runtime/oti/j9javaaccessflags.h

```diff
@@ -167,4 +165,8 @@
 #define J9StaticFieldRefFlagBits 0x3F
 #endif /* defined(J9VM_OPT_VALHALLA_FLATTENABLE_VALUE_TYPES) */
 
+/* ImplicitCreation attribute flags */
+#define J9AccImplicitCreateHasDefaultValue 0x01
+#define J9AccImplicitCreateNonAtomic 0x20
```

Comment:
Is this intentional ? `J9AccImplicitCreateHasDefaultValue` is `0x1`, and `J9AccImplicitCreateHasDefaultValue` is `0x20`.

Reply:
Thanks for the catch. default was also previously wrong and I corrected it here. I'll update nonatomic as well.